app/src/main/java/com/github/se/eventradar/ui/home/Home.kt

```diff
@@ -126,7 +130,10 @@ fun HomeScreen(
         contentColor = MaterialTheme.colorScheme.primary) {
           Tab(
               selected = selectedTabIndex == 0,
-              onClick = { selectedTabIndex = 0 },
+              onClick = {
+                uiState.tab = Tab.BROWSE
```

Comment:
You will not be able to update the uiState this way, you will have to have a helper function with access to `_uiState` to change it.